src/utils/graphql.ts

```diff
@@ -0,0 +1,59 @@
+import gql from 'graphql-tag';
+
+export const BOATTYPES_BERTHS_QUERY = gql`
```

Comment:
Shouldn't this be `BOAT_TYPES_BERTHS_QUERY`?

Reply:
@Nizar-Rahme